### PR TITLE
Add a unit test for `SiStripHashedDetId`

### PR DESCRIPTION
**Pull request by mmusich**
#### PR description:

Currently `SiStripHashedDetId` is not used by any production workflow but there is a plan to use it in the future. 
Protecting here the functionality with a unit test.

#### PR validation:

added unit tests `scram b runtests_testSiStripHashedDetId` runs fine.

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

N/A

### CODE REVIEW

**Comment by mmusich** [comment]
please test

**Comment by tvami** [comment]
+1
 * new unit test passed https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-12530c/32958/unitTests/src/CalibFormats/SiStripObjects/test/testSiStripHashedDetId/testing.log

**Comment by perrotta** [comment]
+1

File: CalibFormats/SiStripObjects/test/testSiStripHashedDetId.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+ function die { echo $1: status $2; exit $2; }
+
+if [ "${SCRAM_TEST_NAME}" != "" ] ; then
+  mkdir ${SCRAM_TEST_NAME}
+  cd ${SCRAM_TEST_NAME}
+fi
```
Comment (makortel) [inline CalibFormats/SiStripObjects/test/testSiStripHashedDetId.sh:8]: @smuzaffar Do I remember correctly that nowadays SCRAM does effectively the same automatically?

**Pull request merged**